### DISCUSSION TITLE
Working with Dynamic data and multiple locations in booking form

### DISCUSSION THREAD

**Discussion by TommsNL** [liedekef/events-made-easy#603] [Q&A] [answered]
Hello,

I'd like to create a booking form which uses **Dynamic data**

I have multi priced event: 42||47||52||62||0

The first 4 Seats are for registering kids to the event. The booker (parent) can choose a price (the higher prices are like donations).

The last Seat (SEATS{5}/0 price) is to register one extra parent (that are helping at the event).

Is it possible to show the #_DYNAMICDATA in the form on multiple locations? 

For example see the booking form below. In this example below the #_DYNAMICDATA is rendered at each location (probably by design). But is it also prossible to create multipe types of #_DYNAMICDATA?

****Introtext**
**Register kids****

€42,- (Basisbedrag) #_SEATS{1}
€47,-  (€5,- donatie) #_SEATS{2}
€52,-  (€10,- donatie) #_SEATS{3}
€62,-  (€20,- donatie) #_SEATS{4}

#_DYNAMICDATA

**Register Booking data****

**Register extra perent?****
#_SEATS{5}

#_DYNAMICDATA

**Comment by liedekef** [comment, marked as answer]
Using #_DYNAMICDATA multiple times is not supported (it will work, but won't do what you expect later on). I'll try to avoid rendering it multiple times.
In short: no, it is not possible to use different types of #_DYNAMICDATA. The code is already very complicated as is ... You can always use jquery to show/hide cerrtain fields if you want (but that's generic html-knowledge, not EME specific).

**Reply by liedekef** [reply]
Btw: it is possible to code this up, but this requires quite a bit of coding and won't be done without proper support :-)

**Reply by TommsNL** [reply]
Ok thanks for the reply @liedekef For now I will try to structure my booking form in a way that I only need to generate #_DYNAMICDATA  at the end.
